src/node/desktop/src/ui/widgets/choose-r/preload.ts

```diff
@@ -97,8 +97,9 @@ const callbacks: Callbacks = {
     ipcRenderer.send('cancel');
   },
 
-  browse: () => {
-    ipcRenderer.send('browse');
+  browse: async () => {
+    const shouldCloseDialog = await ipcRenderer.invoke('browse-r-exe');
```

Comment:
Is there a reason for changing the name here? (Do we need something like namespacing for the names used in IPC requests / responses?)

Reply:
Well... `browse` is a very common name. My fear was that this would be used somewhere else and mess up with our code. So I decided to change it already not to lose any time or have headaches in the future.